src/ui.container-accordion/useAttrs.ts

```diff
@@ -1,10 +1,21 @@
 import { computed } from "vue";
 import useUI from "../composables/useUI.ts";
 
-import defaultConfig from "./config.js";
+import defaultConfig from "./config.ts";
 
-export default function useAttrs(props, { isOpened }) {
-  const { config, getKeysAttrs, hasSlotContent, getExtendingKeysClasses } = useUI(
+import type { Ref } from "vue";
+import type { UseAttrs } from "../types.ts";
+import type { UAccordionProps, Config } from "./types.ts";
+
+type ActiveState = {
```

Comment:
`ActiveState` >>> `ComponentState` - use same general type name for all components